nifi-nar-bundles/nifi-spark-bundle/nifi-livy-processors/src/test/java/org/apache/nifi/processors/livy/TestExecuteSparkInteractive.java

```diff
@@ -85,16 +85,17 @@ private static TestServer createServer() throws IOException {
 
     @Test
     public void testSparkSession() throws Exception {
-
         addHandler(new LivyAPIHandler());
 
-        runner.enqueue("print \"hello world\"");
+        String code = "print \"hello world\" //'?!<>[]{}()$&*=%;.|_-\\";
```

Comment:
instead of changing the existing UT, what about creating a new one for this specific case? It is good that every UT has very little scope so that failures can clearly indicate to the developer what he/she broke applying a patch...

Reply:
Thanks for the review @mgaido91. I made a minor refactor to the tests and separated the cases.